### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by grouma**
These warnings will cause issues with Google3. Upgrading the warnings to errors and resolving the issues.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.12.23+1
```
Comment (nex3) [inline CHANGELOG.md]: It's not worth publishing this version, since it doesn't have any user-visible changes.

File: analysis_options.yaml [hunk 2]
```diff
@@ -1,5 +1,13 @@
 analyzer:
   strong-mode: true
+errors:
+  unused_element: error
+  unused_import: error
+  unused_local_variable: error
+  dead_code: error
```
Comment (nex3) [inline analysis_options.yaml]: Why are you making these errors?

File: CHANGELOG.md (hunk 1 above)
Comment (grouma) [inline CHANGELOG.md]: Discussed offline. Necessary for a Google3 sync.

File: analysis_options.yaml (hunk 2 above)
Comment (grouma) [inline analysis_options.yaml]: Added a comment. Necessary to not break Google3.

**Pull request merged**